src/grain/grain.test.js

```diff
@@ -81,4 +77,36 @@ describe("src/grain/grain", () => {
       }
     });
   });
+
+  describe("multiplyFloat", () => {
+    it("behaves reasonably for tiny grain values", () => {
+      // $ExpectFlowError
+      expect(multiplyFloat(1n, 5)).toEqual(5n);
+    });
+    it("behaves reasonably for larger grain values", () => {
+      // $ExpectFlowError
+      expect(multiplyFloat(ONE, 2)).toEqual(2n * ONE);
+    });
+    it("has small error on large grain values", () => {
+      // To compare with arbitrary precision results, see:
+      // https://observablehq.com/@decentralion/grain-arithmetic
+
+      // Within 1 attoGrain of "true" value
+      // $ExpectFlowError
+      expect(multiplyFloat(ONE, 1 / 1337)).toEqual(747943156320119n);
+
+      // Within 300 attoGrain of "true" value
+      // $ExpectFlowError
+      expect(multiplyFloat(ONE, Math.PI)).toEqual(3141592653589793280n);
```

Comment:
It’s a bit weird that if we were to *improve* the error bounds on
`multiplyFloat` (per my comment above) that these tests could fail.
Maybe

```javascript
const actual = multiplyFloat(ONE, 1 / 1337);
const expected = 3141592653589793000n;
const tolerance = 300000n;
expect(actual).toBeGreaterThanOrEqual(expected - tolerance);
expect(actual).toBeLessThanOrEqual(expected + tolerance);
```

such that they pass as written and we can tighten if desired?

Reply:
Second this, and would suggest resembling the number counterpart jest has.
`.toBeCloseTo(number, numDigits?)`
https://jestjs.io/docs/en/expect#tobeclosetonumber-numdigits

Maybe
`expectBigintToBeCloseTo(actual, expected, tolerance)`

Reply:
I saw `toBeCloseTo`, but it’s documented as taking only `number`s, while
other assertions on the page take `number | bigint`, so I wouldn’t trust
`toBeCloseTo` to be lossless. That’s why I suggested what I did, which
I think works fine:
<https://github.com/sourcecred/sourcecred/commit/6fc1a8efe9550b266aa570347e12c2ecdd492e63#diff-dac25656aec5c30832a9dadf8c52ff54R104-R108>

Reply:
I'll let you fix this when we fix the precision.